conf/modules/sentieon_haplotyper.config

```diff
@@ -45,14 +45,16 @@ process {
         ]
     }
 
-    withName: '.*BAM_VARIANT_CALLING_SENTIEON_HAPLOTYPER:VCF_VARIANT_FILTERING_GATK:FILTERVARIANTTRANCHES' {
-        ext.prefix       = {"${meta.id}.haplotyper"}
-        ext.args         = { "--info-key CNN_1D" }
-        publishDir       = [
-            mode: params.publish_dir_mode,
-            path: { "${params.outdir}/variant_calling/sentieon_haplotyper/${meta.id}/"},
-            pattern: "*{vcf.gz,vcf.gz.tbi}"
-        ]
+    if (params.tools && params.tools.contains('sentieon_haplotyper')) {
```

Comment:
It the if-clause added to avoid the standard WARN when `BAM_VARIANT_CALLING_SENTIEON_HAPLOTYPER:VCF_VARIANT_FILTERING_GATK:FILTERVARIANTTRANCHES`  doesn't run?

Reply:
yes